.github/workflows/build.yaml

```diff
@@ -189,6 +189,32 @@ jobs:
           command: doc
           args: --release --locked --no-deps --features sim
 
+  coverage:
+    runs-on: ubuntu-20.04
+    needs:
+      - "rustfmt"
+      - "markdown-lint"
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./.github/actions/sgxsdk
+      - uses: ./.github/actions/dcap-libs
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          # FIXME: replace with stable ASAP
+          toolchain: nightly-2022-07-22
+          override: true
+          components: llvm-tools-preview
+      - uses: taiki-e/install-action@cargo-llvm-cov
+      - uses: actions-rs/cargo@v1
+        with:
+          command: llvm-cov
+          # The FFI bindings crates always report 100% because they have no code
+          args: --features sim --ignore-filename-regex '\/sys/src/lib\.rs' --workspace --lcov --output-path lcov.info
```

Comment:
:question: Looking at https://app.codecov.io/gh/mobilecoinfoundation/sgx/blob/216823bd78d35cacd490bdae76cd005bf9032dc6/core/build/src/lib.rs
Can we think of a good way to test `mc-sgx-core-build` or should it be excluded too?

Reply:
I think we can have unit tests around it, but I'd like to put those on `mc-util-build-script::Environment` and friends instead. If we had a generic "give me the contents of this env var" and the specific "append this string" stuff, we can certainly test the cases w/ a custom runner framework (i.e. maybe a set of env file test cases used as test matrix) 

That's complex enough that I'd like to pull `mc-util-build-script` into it's own repo first and do them there, once.